src/config.rs

```diff
@@ -108,6 +108,10 @@ pub struct Config {
     #[structopt(long = "rsync-disable")]
     pub rsync_disable: bool,
 
+    /// Support different rsync base URIs, include host and module: <rsync_dir>/current/<host>/<module>/..
```

Comment:
Which use case does this enable?

Reply:
https://github.com/NLnetLabs/krill-sync/issues/18

RIPE NCC in particular are using a single RRDP repository for multiple rsync modules (one for the TA, one for the rest).